release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1590,15 +1590,41 @@ Previously, the image pruner stopped when it failed to delete an image. As a res
+
+*  Previously, the `oc adm catalog mirror` command could not be easily used to mirror Operator catalogs across an airgapped environment. With this enhancement, the contents of a catalog can be mirrored to a file system, placed onto removable media, and then mirrored back from the file system to a registry for usage by an airgapped cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1919168[*BZ#1919168*])
+
+* The Catalog Operator previously created bundle unpacking jobs for an install plan without setting a timeout. In the case of a non-existent or deleted bundle image, this caused the job would to running forever and the install plan would stay in the `Installing` phase with no indication of the job's pod failing to resolve the image. With this fix, the Catalog Operator now sets a default `10m` timeout on bundle unpacking jobs, which can be configured by using `--bundle-unpack-timeout` flag. As a result, bundle unpacking jobs fail after the configured timeout, and the installation also transitions to a `Failed` phase with the reason visible in `status.conditions` and `status.bundleLookups.conditions` properties. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1921264[*BZ#1921264*])
+
+* Operators that were installed on clusters prior to {product-title} 4.6 were previously not identified as coming from a given Operator package for the purposes of dependency resolution and upgrade selection. This caused existing Operator installations to conflict with the criteria of their own subscription, which blocked upgrades and dependency resolution within the namespace. This fix updates OLM to infer the package name and version for Operators that are referenced by a subscription. As a result, upgrades and dependency resolution proceed as expected in this scenario. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1921953[*BZ#1921953*])
+
+* The `Info` log level used for transient errors caused verbose OLM Operator logs for the default configuration. This fix changes the
+transient error log level to debug. As a result, fewer non-critical logs are visible for the debug configuration. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1925614[*BZ#1925614*])
+
+* Previously, the `subscription.spec.config.resources` section was always applied to the installed deployment, even when it was unset or empty. This caused resources defined in the cluster service version (CSV) to be ignored, and only the resources defined in the `subscription.spec.config.resources` section were used. This fix updates OLM to override deployment-specific resources only when the `subscription.spec.config.resources` section is set to a non-nil or non-empty value. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1926893[*BZ#1926893*])
```

Comment:
the  _`subscription.spec.config.resources` section_  of what? Is that important to mention or will folks understand?

Again, _only when the `subscription.spec.config.resources` section_ of what?

Reply:
Changed to:

> the `spec.config.resources` section of a `Subscription` object